client/blocks/app-promo/index.jsx

```diff
@@ -1,55 +1,85 @@
 import { Card, Button } from '@automattic/components';
 import { useLocalizeUrl } from '@automattic/i18n-utils';
+import classNames from 'classnames';
 import { useTranslate, useRtl } from 'i18n-calypso';
 import wpToJpImageRtl from 'calypso/assets/images/jetpack/wp-to-jp-rtl.svg';
 import wpToJpImage from 'calypso/assets/images/jetpack/wp-to-jp.svg';
+import QrCode from 'calypso/blocks/app-promo/qr-code';
 import AppsBadge from 'calypso/blocks/get-apps/apps-badge';
 import CardHeading from 'calypso/components/card-heading';
 import userAgent from 'calypso/lib/user-agent';
-
+import { useSelector } from 'calypso/state';
+import { isUserLoggedIn } from 'calypso/state/current-user/selectors';
 import './style.scss';
 
-export const AppPromo = () => {
+export const AppPromo = ( {
+	title = false,
+	iconSize = 29,
+	source = 'calypso-customer-home',
```

Comment:
Could the default just be nothing or "other", to avoid accidentally assigning source when using the component?

Reply:
Will update it to `calypso-app-promo`. a bit more specific.